keyserver/server.go

```diff
@@ -21,21 +21,34 @@ import (
 	"github.com/coniks-sys/coniks-go/utils"
 )
 
+// A ServerConfig contains configuration values
+// which are read at initialization time from
+// a TOML format configuration file.
 type ServerConfig struct {
-	configFilePath      string
-	DatabasePath        string          `toml:"database"`
+	// DatabasePath is a path to the database for storing the server's directory.
+	DatabasePath string `toml:"database"`
+	// LoadedHistoryLength is the maximum length of current snapshots stored in memory.
 	LoadedHistoryLength uint64          `toml:"loaded_history_length"`
 	Policies            *ServerPolicies `toml:"policies"`
 	TLS                 *TLSConnection  `toml:"tls"`
+	configFilePath      string
```

Comment:
Maybe not quite like that, maybe something more like "Policies contains the server's CONIKS policies configuration"?